code/model/modelrender.h

```diff
@@ -296,8 +296,8 @@ class model_draw_list
 	void reset();
 };
 
-void model_render_immediate(model_render_params *render_info, int model_num, matrix *orient, vec3d * pos, int render = MODEL_RENDER_ALL, bool sort = true);
-void model_render_queue(model_render_params *render_info, model_draw_list* scene, int model_num, matrix *orient, vec3d *pos);
+void model_render_immediate(model_render_params* render_info, int model_num, matrix* orient, vec3d* pos, int render = MODEL_RENDER_ALL, bool sort = true, vec3d show_ship_pos = vmd_zero_vector, bool is_using_showship = false);
```

Comment:
This is my main problem with these changes. It adds additional parameters to a very common function that are only needed for one specific use case. In my opinion, it would be better to fix the underlying issue that the view position is not set correctly instead of taking this shortcut.